src/OpenLoco/Vehicles/VehicleHead.cpp

```diff
@@ -1167,8 +1167,12 @@ namespace OpenLoco::Vehicles
 
             status = Status::waitingAtSignal;
 
+            auto newConfig = Config::getNew();
```

Comment:
This will copy the config struct. Use a reference:
```suggestion
            auto& newConfig = Config::getNew();
```

Reply:
My bad. Fixed.